src/main/resources/l10n/JabRef_it.properties

```diff
@@ -1727,3 +1700,24 @@ Online_help_forum=
 Custom=
 
 Converts_HTML_code_to_Unicode.=
+Connection=
+Host=
+Port=
```

Comment:
If possible it would be nice to reuse the existing translations of Port and Database. (One can consider reusing "Server hostname" and possibly "Username" as well, to not add more strings.)